src/Microsoft.TestPlatform.CrossPlatEngine/Client/Parallel/ParallelProxyExecutionManager.cs

```diff
@@ -197,22 +197,17 @@ public void Close()
                 return true;
             }
 
-            // In case of DataCollection we only start dc.exe on initialize, & close once the TestRun is complete,
-            // So instead of resuing ProxyExecutionManager, we will close it here, & create new PEMWDC
-            // In Case of Abort, clean old one and create new proxyExecutionManager in place of old one.
-            if (!this.SharedHosts || testRunCompleteArgs.IsAborted || (proxyExecutionManager is ProxyExecutionManagerWithDataCollection))
```

Comment:
Why did we remove these checks? were they bogus?

Reply:
Earlier they were needed only for those scenarios, and in other scenarios they were reusing the execution manager. But since the test host dies it's not good to have that around. and was observing an exception, which tries to send message over a channel that's already closed.